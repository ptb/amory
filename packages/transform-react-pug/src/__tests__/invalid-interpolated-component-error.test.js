import {testCompileError} from './test-helper';

testCompileError(__dirname + '/invalid-interpolated-component-error.input.js');
