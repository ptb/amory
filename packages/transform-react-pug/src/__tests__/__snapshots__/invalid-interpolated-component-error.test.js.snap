// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect an error to be thrown 1`] = `
"<basedir>/src/__tests__/invalid-interpolated-component-error.input.js: <basedir>/src/__tests__/invalid-interpolated-component-error.input.js:5
    3| module.exports = pug\`
    4|   div
  > 5|     \${name} hello hello
    6| \`;
    7| 

Only components can have children and attributes"
`;
